e2e/e2e_test.go

```diff
@@ -32,6 +37,14 @@ func init() {
 	fmt.Println("timeout for deploytimeout ", deployTimeout)
 }
 
+func setDefaultKubeconfig() {
+	_, exists := os.LookupEnv("KUBECONFIG")
+	if !exists {
+		defaultKubeconfig := filepath.Join(os.Getenv("HOME"), ".kube", "config")
+		os.Setenv("KUBECONFIG", defaultKubeconfig)
```

Comment:
as we are setting this ENV now do we need to unset this ENV after finishing testing?

Reply:
instead of making code changes can we update the readme to set the KUBECONFIG env before running e2e?

Reply:
I don't think so. The ENV variables are only set for the current process and it's child processes. So if e2e tests is ended, the variable is gone.

Reply:
> instead of making code changes can we update the readme to set the KUBECONFIG env before running e2e?

I think we can just remove the kubeconfig default option. but all other flags are having default option and most of user do not want to modify kubectl config path.

Reply:
as the env will be set in the current process and also this code honors the e2e readme documentation, am fine with the changes